ChainSafe/web3.unity
# Marketplace package

Moved IPFS & marketplace core functionality into a package to modularize it's use and left a placeholder for a use case implementation for 3.0.

To Test:

Run setup.sh
Check that the sdk functions as usual without the marketplace package.
Install the marketplace package locally and check that the marketplace/ipfs functions work there too.
The marketplace calls will enable in the sample scene similar to how ramp does once the package is present.
